features/freaks/src/main/kotlin/com/agilefreaks/freaks_catalog/features/freaks/DetailsViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package com.agilefreaks.freaks_catalog.features.freaks
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.agilefreaks.freaks_catalog.features.freaks.repository.FreakDetailsRepository
+import kotlinx.coroutines.launch
+
+class DetailsViewModel(private val repository: FreakDetailsRepository) : ViewModel() {
+    private var _curFreak = MutableLiveData<FreakD>()
+    val curFreak: LiveData<FreakD>
+        get() = _curFreak
```

Comment:
A better name for this can be "freak" because here we don't have a list of freaks. We have only one freak